index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Capiston-project-1</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <nav>
+          <div class="navs">
+            <div class="top-nav">
+              <a href="#"><img class="top-item" src="assets/facebook.png" alt="facebook"></a>
+         <a href="#"><img class="top-item" src="assets/twitter.png" alt="twitter"></a>
+               <a href="#" class="top-item">English</a>
+                <a href="#" class="top-item">My Page</a>
+                <a href="#"class="top-item">Logout</a>
+            </div>
+            <div class="bottom-nav">
+              <img class="nav-logo" src="assets/logo.jpg" alt="logo">
+            <div class="nav-links">
+                <a class="nav-link" href="about.html">About</a>
+                <a class="nav-link" href="#partner">Partner</a>
+                <a class="nav-link" href="#">Join</a>
+                <a class="nav-link" href="#">Sponsor</a>
+                <a class="nav-link" id="kbu" href="#">CC-CAMPAIN</a>   
+            </div>
+          </div>
+                    <!--  mobile-menu-->
+                     <header class="head-mob">
+                    <nav class="nav-mob">
+                        <div class="menu-links">
+                            <img class="logo" src="assets/logo.jpg" alt="logo">
```

Comment:
- [ OPTIONAL] The logo should link to the home page so that when users click on it, the home page is loaded. This improves user experience 🛩️